glfs.c

```diff
@@ -479,9 +479,7 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 static int tcmu_glfs_open(struct tcmu_device *dev)
 {
```

Comment:
Patch looks ok to me.

Just a question. For glfs, if a request is sent that is past the end of the device what error code do you get in glfs_async_cbk?

Reply:
I think EIO, but I will have to read the code to confirm. Will reply here after verifying tomorrow ?

Reply:
@mikechristie There won't be any error. Write request on the block device is converted to write request on a file on glusterfs, so write will be successful. May I know what is the behaviour you would like to see?

Reply:
I was not looking for any specific behavior to be implemented. I was just wondering what will happen for if/when we start to add thin provisioning features on the SCSI side.